FlavorTrack/Core/Business List/BusinessListVC.swift

```diff
@@ -56,42 +56,49 @@ private extension BusinessListVC {
 		collectionView.register(BusinessCell.self, forCellWithReuseIdentifier: BusinessCell.reuseIdentifier)
 		collectionView.delegate = self
 	}
-	
+
 	private func configDataSource() {
 		dataSource = .init(collectionView: collectionView,
 						   cellProvider: { (collectionView, indexPath, business) -> UICollectionViewCell? in
-			let cell = collectionView.dequeueReusableCell(withReuseIdentifier: BusinessCell.reuseIdentifier, for: indexPath) as! BusinessCell
+			// swiftlint:disable force_cast
```

Comment:
Rather than disabling fore_cast here, let's do optional binding. Like this:
`guard let cell = collectionView.dequeueReusableCell(
				withReuseIdentifier: BusinessCell.reuseIdentifier,
                for: indexPath) as? BusinessCell else {
                    fatalError("BusinessCell does not exisst")
                }`